tests/unit/classifier/svm/NewtonSVM_unittest.cc

```diff
@@ -0,0 +1,87 @@
+/*
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Authors: Shubham Shukla
+ */
+#include <functional>
+#include <rxcpp/rx-lite.hpp>
+#include <shogun/lib/Signal.h>
+
+#include "environments/LinearTestEnvironment.h"
+#include <gtest/gtest.h>
```

Comment:
pls include gtest.h first, they use some magic macros that cause problems with some compilers otherwise